src/springfield/fields.py

```diff
@@ -479,6 +542,7 @@ def jsonify(self, value):
     int: IntField(),
     basestring: StringField(),
     unicode: StringField(),
+    bytes: BytesField(),
```

Comment:
@six8 Another option: delete `get_field_for_type` given it has no tests, no usage, and dubious usefulness.